mypy.ini

```diff
@@ -5,7 +5,6 @@ disallow_any_unimported = True
 no_implicit_optional = True
 check_untyped_defs = True
 warn_return_any = True
-warn_unused_ignores = True
```

Comment:
why have you removed this?

Reply:
Coz currently mypy causes an error when you type ignore a whole file. It's an open issue. I needed to ignore `[attr-defined]` for the entire test_view file